lib/service_actor/result.rb

```diff
@@ -51,5 +51,28 @@ def [](name)
     def display
       to_h.fetch(:display)
     end
+
+    def respond_to?(method_name, include_private = false)
+      !!method_missing(method_name) || super
+    end
+
+    private
+
+    def method_missing(symbol, *args)
+      attribute = symbol.to_s.chomp("?")
+
+      if symbol.to_s.end_with?("?") && respond_to?(attribute)
+        define_singleton_method symbol do
+          attribute_value = send(attribute.to_sym)
+
+          # Same as ActiveSupport’s #present?
+          !(attribute_value.respond_to?(:empty?) ? !!attribute_value.empty? : !attribute_value)
```

Comment:
Maybe we can move the `!` around to simplify a tad more:

```suggestion
          attribute_value.respond_to?(:empty?) ? !attribute_value.empty? : !!attribute_value
```